sam/template.yaml

```diff
@@ -0,0 +1,47 @@
+AWSTemplateFormatVersion: '2010-09-09'
```

Comment:
This file is not referenced anywhere in the docs. Is it only required for validation?

Reply:
it is a config file for the aws sam cli. https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification-template-anatomy.html

Sam can deploy the app on AWS.

It contains among other the description that will be displayed in the serverless application repository, so it would be great if you could take a close look ;-)

Reply:
Yes, meanwhile I was reading it. Looks good!